lib/utility/parent-port.ts

```diff
@@ -3,27 +3,28 @@ import { MessagePortMain } from '@electron/internal/browser/message-port-main';
 const { createParentPort } = process._linkedBinding('electron_utility_parent_port');
 
 export class ParentPort extends EventEmitter {
-  #port: any
+  #port: ParentPort
   constructor () {
     super();
     this.#port = createParentPort();
-    this.#port.emit = (channel: string, event: {ports: any[]}) => {
+    this.#port.emit = (channel: string | symbol, event: {ports: any[]}) => {
```

Comment:
also surprised this doesn't lint to

```suggestion
    this.#port.emit = (channel: string | symbol, event: { ports: any[] }) => {
```

are we 100% sure lint is running properly against these files..? maybe my brain linter is just broken

Reply:
These files are definitely linted and we do have `object-curly-spacing` enabled but it didn't warn for this declaration. Another missed location https://github.com/electron/electron/blob/8212616c761d95c9088770fbe6a05615a53cff4f/lib/browser/message-port-main.ts#L8. Maybe a bug in ESLint ?

Reply:
Refs https://github.com/electron/electron/pull/34980/commits/f6b160abb490742736220374ef703bc6015a102b